fitbenchmarking/controllers/levmar_controller.py

```diff
@@ -90,6 +90,22 @@ def fit(self):
         # self._info[5] = number of `jacf` evaluations
         # self._info[6] = number of linear system solved
 
+    def eval_wrapper(self, params, **kwargs):
```

Comment:
Can you rename this to be something with "j" or "jac" or "jacobian" in the name to make it a bit clearer what it's evaluating. I think this can also be a protected/private function with an underscore.